assignment4.txt

```diff
@@ -0,0 +1,10 @@
+Ava N.
+July 5, 2016
+
+1. Character arrays are arrays of characters. For example: char c='a';. Strings are arrays of characters. For example: char[10]="Hello World";. What this will do is assign char[0]='H', char[1]='e', char[2]='l', char[3]='l', char[4]='o', char[5]=' ', char[6]='W', char[7]='o', char[8]='r', char[9]='l', char[10]='d'. String assignment is only at declaration. Also, at the beginning of the program should use "#include <string.h>". 
```

Comment:
Hm, not exactly - a string is basically a character array, but with a NULL terminator as the last element.